data/update_script.sh.erb

```diff
@@ -85,6 +87,22 @@ if ! /tmp/pre_script.sh ; then
   exit 1
 fi
 
+if $PULLPREVIEW_FIRST_RUN ; then
+  if [ -f "$FIRST_RUN_SCRIPT" ]; then
+    echo "Executing $FIRST_RUN_SCRIPT"
+    chmod +x $FIRST_RUN_SCRIPT
+    source $FIRST_RUN_SCRIPT
```

Comment:
Any reason to `source`, instead of executing the script? I'd prefer if those scripts don't cause side-effects in the current environment.